### PR TITLE
Update EgyBestProvider.kt

### PR DESCRIPTION
**Pull request by aymen66**

### CODE REVIEW

**Comment by LagradOst** [comment]
@ImZaw

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt [hunk 1]
```diff
@@ -218,7 +218,8 @@ class EgyBestProvider : MainAPI() {
         val jsonArray = parseJson<List<Sources>>(requestJSON)
         for (i in jsonArray) {
             val quality = i.quality
-            val link = i.link
+            val link = i.link//m3u8
+            val downloadlink = i.mp4//mp4
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt:222]: This variable is unused :confused:

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt [hunk 2]
```diff
@@ -47,9 +47,9 @@ class EgyBestProvider : MainAPI() {
         val pages = arrayListOf<HomePageList>()
         doc.select("#mainLoad div.mbox").apmap {
             val name = it.select(".bdb.pda > strong").text()
-            if (it.select(".movie").first()?.attr("href")?.contains("season-(.....)|ep-(.....)".toRegex()) == true) return@apmap
+            if (it.select("a.movie").first()?.attr("href")?.contains("season-(.....)|ep-(.....)".toRegex()) == true) return@apmap
```
Comment (Blatzar) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt:50]: Do these changes do anything? Homepage seems to work fine

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt (hunk 1 above)
Comment (aymen66) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt:222]: i forgot to comment it

**Comment by ImZaw** [comment]
> @ImZaw

I don’t really see something changes or a fix in that pull request

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt (hunk 2 above)
Comment (ImZaw) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt:50]: nope, it won’t change anything

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt (hunk 1 above)
Comment (ImZaw) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/EgyBestProvider.kt:222]: so you just added the mp4 link?

**Pull request closed without merging**